app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError, NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    mask_for_friends = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
```

Comment:
Returning immediately upon encountering a `VaccineError` means not all friends are checked. According to the previous review, you should accumulate errors and provide a summary after checking all friends. Consider using a flag or counter to track vaccination status and return a summary after the loop.